internal/flags/flags.go

```diff
@@ -26,7 +26,7 @@ const (
 	RegionShort                     = "r"                               // RegionShort flag
 	Members                         = "members"                         // Members flag
 	MembersShort                    = "m"                               // MembersShort flag
-	InstanceSize                    = "instanceSize"                    // InstanceSize flag
+	ClusterTier                     = "clusterTier"                     // ClusterTier flag
```

Comment:
if you check the comments in the jira ticket we agreed to make this just `tier` given you are in a cluster command
also from the comments, this was instanceSize instead of just size because we have diskSize flag